public/context.json

```diff
@@ -35,7 +36,10 @@
     "title": {"@id": "dcterms:title"},
     "abstract": {"@id": "dcterms:abstract"},
 
+
     "edition": {"@id": "bf:editionStatement"},
+    "projection": {"@id": "bf:cartographicProjection"},
+    "scale": {"@id": "bf:cartographicScale"},
```

Comment:
So in the rdf-vocab gem the range is listed as [literal](https://github.com/ruby-rdf/rdf-vocab/blob/731d4d43ad5819a4a412893d062ca4426a10c92e/lib/rdf/vocab/bibframe.rb#L488). Could we still get away with using it?